scripts/install-lansa-base.ps1

```diff
@@ -214,28 +214,28 @@ try
 
     # GUI Application Installation
     if ( $Cloud -ne "Docker" ) {
-        Run-ExitCode 'choco' @( 'install', 'googlechrome', '-y', '--no-progress' ) | Write-Host
+        Run-ExitCode 'choco' @( 'install', 'googlechrome', '-y', '--no-progress','--ignore-checksums', '-s choco' ) | Write-Host
```

Comment:
Why have you specified `--ignore-checksums`?

If they are not matching then we need to use an older version which DOES have valid checksums. That is one of the beauties of having our own chocolatey server is that we can be assured that the checksums DO match.

So, find the last Googlechrome  install that works and I'll upload it and delete the one thats there currrently.

Reply:
hi @robe070
cc: @ravicharan07 
Please refer to the note section in the SS below. It suggests that we might ignore the checksum mismatch. Please let us know your comments on this one.

> This package always installs the latest version of Google Chrome, regardless of the version specified in the package. Google does not officially offer older versions of Chrome for download. Because of this, you may get checksum mismatch between the time Google releases a new installer, and the package is automatically updated.

![image](https://user-images.githubusercontent.com/24430923/99642371-70f97600-2a71-11eb-9faa-085ddb9effc4.png)

Reply:
I see. So its not possible for us to download a complete package because the install itself always comes from Google. Its separate from the Chocolatey package.  We cannot fix the install version. And if thats the case then the only solution is to ignore checksums.